stage1/init/common/pod.go

```diff
@@ -197,13 +197,35 @@ func SetJournalPermissions(p *stage1commontypes.Pod) error {
 		return errwrap.Wrap(errors.New("error creating journal dir"), err)
 	}
 
-	a, err := acl.Parse(fmt.Sprintf("g:%d:r-x,m:r-x", rktgid))
+	a, err := acl.InitACL()
 	if err != nil {
-		log.PanicE("error parsing ACL string", err)
+		return err
 	}
 	defer a.Free()
 
-	if err := a.SetFileDefault(journalPath); err != nil {
+	if err := a.ParseACL(fmt.Sprintf("g:%d:r-x,m:r-x", rktgid)); err != nil {
+		return fmt.Errorf("error parsing ACL string: %v", err)
+	}
+
+	val, err := a.Valid()
+	if err != nil {
+		return err
+	}
+	if !val {
```

Comment:
Not sure if I understand. Shouldn't we bail out if `ParseACL` fails or makes whatevel acl invalid? I'm a noob wrt acl, so I was wondering what's the point of checking validity here.